sdk/monitor/azure-monitor-opentelemetry-exporter/src/samples/java/com/azure/monitor/opentelemetry/exporter/ReadmeSamples.java

```diff
@@ -44,8 +44,7 @@ public void setupExporter() {
         // Create Azure Monitor exporter and initialize OpenTelemetry SDK
         // This should be done just once when application starts up
         AutoConfiguredOpenTelemetrySdkBuilder sdkBuilder = AutoConfiguredOpenTelemetrySdk.builder();
-
-        new AzureMonitorExporterBuilder()
```

Comment:
could we an example that shows how to customize collection further? E.g. add a processor or another exporter (e.g. console), add view, etc?